reference/admin.yaml

```diff
@@ -7709,6 +7709,158 @@ components:
+          description: This represents the id of the expense policy rule.
+        description:
+          allOf:
+            - $ref: '#/components/schemas/description'
+          description: |
+            This represents the description of the expense policy rule.
+          example: Receipt is mandatory for expense above $50
+        action_show_warning:
+          type: boolean
+          description: >
+            If it is set to true, then a warning is shown to the employee,
+            whenever he attempts to create an expense that violates this expense
+            policy rule.
+          example: true
+    expense_policy_violating_expenses_query_params:
+      type: object
+      description: >
+        Query params that can be combined and used to fetch the violating
+        expenses.
+      properties:
+        start_date:
+          type: string
+          description: |
+            Spent start date from where the expenses should be fetched.
+          example: '2021-12-01'
+        end_date:
+          type: string
+          description: |
+            Spent end date till where the expenses should be fetched.
+          example: '2021-12-01'
+        state:
+          type: string
+          description: |
+            State of the expenses.
+          example: neq.DRAFT
+        category_id:
+          type: string
+          description: |
+            Category ids of the expenses.
+          example: in.(60506)
+        project_id:
+          type: string
+          description: |
+            Project ids of the expenses.
+          example: in.(261155)
+        cost_center_id:
+          type: string
+          description: |
+            Cost center ids of the expenses.
+          example: in.(7475)
+        currency:
+          type: string
+          description: |
+            Currency of the expenses.
+          example: USD
+        vehicle_type:
+          type: string
+          description: |
+            Vehicle type of the expenses.
+          example: ELECTRIC_CAR
+        employee_id:
+          $ref: '#/components/schemas/employee_id'
+    expense_policy_individual_desired_states:
+      type: object
+      properties:
+        rule:
+          $ref: '#/components/schemas/expense_policy_rule_minimal'
+        amount:
+          allOf:
+            - $ref: '#/components/schemas/amount'
+          description: |
+            Amount calculated by policy to be applied to the expense.
+          example: 500
+        violating_expenses_query_params:
+          $ref: '#/components/schemas/expense_policy_violating_expenses_query_params'
+    expense_policy_final_desired_state:
+      type: object
+      properties:
+        expense_id:
+          type: string
+          description: |
+            This represents the id of the expense.
+          example: txabsdfSYqd
+        flag:
+          type: boolean
+          description: |
+            If it is set to true, then the expense will be flagged.
```

Comment:
A `true` value here means, that the {policy calculation} suggests this expense be flagged.

Reply:
applicable for all the roles

Reply:
A `true` value here means, that as per {policy calculation} this expense should be flagged.